content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -23,61 +23,60 @@ This document describes the current state of `VolumeSnapshots` in Kubernetes. Fa
 
-A `VolumeSnapshot` is a request for snapshot of a volume by a user. It is similar to a PersistentVolumeClaim.
+A `VolumeSnapshot` is a request for snapshot of a volume by a user. It is similar to a `PersistentVolumeClaim`.
 
-While `VolumeSnapshots` allow a user to consume abstract storage resources, cluster administrators
-need to be able to offer a variety of `VolumeSnapshotContents` without exposing
-users to the details of how those volume snapshots should be provisioned. For these needs
+Cluster administrators need to be able to offer different ways in
+which volume snapshots can be created without exposing
+users to the low level details. For these needs
 there is the `VolumeSnapshotClass` resource.
 
 Users need to be aware of the following when using this feature:
 
-* API Objects `VolumeSnapshot`, `VolumeSnapshotContent`, and `VolumeSnapshotClass` are CRDs, not part of the core API.
+* API Objects `VolumeSnapshot`, `VolumeSnapshotContent`, and `VolumeSnapshotClass` are CRDs, not part of the core API. The CSI drivers that support the volume snapshot functionality will automatically install these required CRDs (*Note:* Once the Snapshot APIs enter beta state, CRDs will not be installed automatically any more).
+
 * `VolumeSnapshot` support is only available for CSI drivers.
 * As part of the deployment process, the Kubernetes team provides a sidecar helper container for the snapshot controller called `external-snapshotter`. It watches `VolumeSnapshot` objects and triggers `CreateSnapshot` and `DeleteSnapshot` operations against a CSI endpoint.
 * CSI drivers may or may not have implemented the volume snapshot functionality. The CSI drivers that have provided support for volume snapshot will likely use `external-snapshotter`.
-* The CSI drivers that support volume snapshot will automatically install CRDs defined for the volume snapshots.
+
 
 ## Lifecycle of a volume snapshot and volume snapshot content
 
-`VolumeSnapshotContents` are resources in the cluster. `VolumeSnapshots` are requests for those resources. The interaction between `VolumeSnapshotContents` and `VolumeSnapshots` follow this lifecycle:
+`VolumeSnapshotContent` objects are resources in the cluster and `VolumeSnapshot` objects are requests for those resources. The interaction between `VolumeSnapshotContent` and `VolumeSnapshot` objects follows this lifecycle:
 
 ### Provisioning Volume Snapshot
 
 There are two ways snapshots may be provisioned: statically or dynamically.
 
 #### Static
-A cluster administrator creates a number of `VolumeSnapshotContents`. They carry the details of the real storage which is available for use by cluster users. They exist in the Kubernetes API and are available for consumption.
+A cluster administrator creates a number of `VolumeSnapshotContent` resources describing snapshots on the underlying storage that are available for use by cluster users. They exist in the Kubernetes API and are available for consumption.
```

Comment:
Emphasize that these are pre-existing snapshots that they want to import into Kubernetes.